homeassistant/components/enphase_envoy/quality_scale.yaml

```diff
@@ -1,31 +1,17 @@
 rules:
   # Bronze
-  action-setup:
-    status: done
-    comment: only actions implemented are platform native ones.
```

Comment:
oh this makes it exempt

(I know you are going to look into adding them, but then we can change it then)